src/virtualenv/seed/wheels/periodic_update.py

```diff
@@ -250,16 +265,20 @@ def _run_do_update(app_data, distribution, embed_filename, for_py_version, perio
     now = datetime.now()
     if periodic:
         source = "periodic"
-        # mark everything not updated manually as source "periodic"
```

Comment:
All the modifications in `_run_do_update` are what lead me to bump the JSON embed format version.